scripts/lint/ourlint.js

```diff
@@ -55,6 +56,33 @@ const TESTS = {
       return `Data file is formatted incorrectly:\n\n${dataDiff}`;
     }
   },
+
+  /* Check redundant trailing slash in URL */
+  checkUrl: (data) => {
+    const hasRedundantTrailingSlash = (url) => {
+      const origin = new URL(url).origin;
+      return url.replace(origin, '') === '/';
```

Comment:
What about this example: `https://example.com//`? It passes but shouldn't. I think that the check could be "only has `/` characters and at least one".

Reply:
Good catch. I can change it's condition to:

```js
return /^\/+$/.test(url.repalce(origin, ''));
```

Reply:
Updated.